internal/runtimehandlerhooks/high_performance_hooks.go

```diff
@@ -275,6 +284,20 @@ func setIRQLoadBalancing(ctx context.Context, c *oci.Container, enable bool, irq
 }
 
 func setCPUQuota(parentDir string, c *oci.Container) error {
+	containerCgroup, containerCgroupParent, systemd, err := containerCgroupAndParent(parentDir, c)
+	if err != nil {
+		return err
+	}
+	podCgroup := filepath.Base(containerCgroupParent)
+	podCgroupParent := filepath.Dir(containerCgroupParent)
+
+	if err := disableCPUQuotaForCgroup(podCgroup, podCgroupParent, systemd); err != nil {
+		return err
+	}
+	return disableCPUQuotaForCgroup(containerCgroup, containerCgroupParent, systemd)
+}
+
+func containerCgroupAndParent(parentDir string, c *oci.Container) (ctrCgroup, parentCgroup string, systemd bool, _ error) {
```

Comment:
There is logic here, how will we make sure it stays in sync with the container creation code?

Reply:
at the very least, the new test case should filter any changes in the cgroup hierarchy that causes this to not work. I'm not sure we can do much more than expect the tests to save us. FWIW I don't expect the hierarchy to change so much it's pretty stable for v1